scripts/irods/test/test_federation.py

```diff
@@ -1777,3 +1819,133 @@ def test_local_zone_user_is_not_allowed_to_delete_delay_rules_created_by_remote_
 
         finally:
             self.local_admin.run_icommand(['iqdel', '-a'])
+
+
+class test_compound_resource_operations(SessionsMixin, unittest.TestCase):
+    def setUp(self):
+        super(test_compound_resource_operations, self).setUp()
+
+        # load federation settings in dictionary (all lower case)
+        self.config = {}
+        for key, val in test.settings.FEDERATION.__dict__.items():
+            if not key.startswith('__'):
+                self.config[key.lower()] = val
+        self.config['local_zone'] = self.user_sessions[0].zone_name
+        if test.settings.FEDERATION.REMOTE_IRODS_VERSION < (4, 0, 0):
+            test.settings.FEDERATION.REMOTE_VAULT = '/home/irods/irods-legacy/iRODS/Vault'
```

Comment:
We don't have a way of testing this AFAIK. Do you feel this is necessary?

Reply:
This was copied from `test_federation.Test_ICommands`. I will just remove it.